_layouts/default.html

```diff
@@ -15,9 +15,6 @@
     <!-- HEADER -->
     <div id="header_wrap" class="outer">
         <header class="inner">
-          {% if site.github.is_project_page %}
```

Comment:
It's not necessary remove this block.

In your site, configure the `_config.yaml` file to set this flag property to `false`.

```yaml
github:
  is_project_page: false
```